woocommerce/woocommerce-android
# Add toolbar to label format options fragment

Part of #10570 

### Description

This PR introduces individual toolbars for LabelFormatOptionsFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Print shipping label -> click on "See label layout and paper size options text") screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="243" alt="Screenshot 2024-02-12 at 6 29 25 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/be50e70c-a7f2-493b-884b-77809246442f">
